plugins/Files/js/components/app.js

```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react'
+import FileBrowser from '../containers/filebrowser.js'
+import AllowanceDialog from '../containers/allowancedialog.js'
+
+const FilesApp = ({unlocked, showAllowanceDialog}) => {
+	let fileBrowserContent
+	if (!unlocked) {
+		fileBrowserContent = (
```

Comment:
not sure how I feel about this pattern. I think I would prefer to stick everything in the final JSX element rather than assigning to a local variable. If you applied that here, the component would be a single expression.